gci/src/main/java/com/danielfireman/gci/Sampler.java

```diff
@@ -0,0 +1,49 @@
+package com.danielfireman.gci;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Sampler {
+	// Default sample rate should be fairly small, so big requests get checked up quickly.
+	private final int DEFAULT_SAMPLE_RATE = 10; // TODO(David) Update this value, if needed
+	// Max sample rate can not be very big because of peaks.
+	// The algorithm is fairly conservative, but we never know.
+	private final int MAX_SAMPLE_RATE = 30; // TODO(David) Update this value, if needed
```

Comment:
Same as above, why is this different than the [go version](https://github.com/gcinterceptor/gci-go/blob/master/gccontrol/sampler.go#L13)?